src/diffusers/loaders.py

```diff
@@ -180,6 +180,7 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
         subfolder = kwargs.pop("subfolder", None)
         weight_name = kwargs.pop("weight_name", None)
         use_safetensors = kwargs.pop("use_safetensors", None)
+        network_alpha = kwargs.pop("network_alpha", None)
```

Comment:
I would also include this in the [`load_lora_weights()` method](https://huggingface.co/docs/diffusers/main/en/api/loaders#diffusers.loaders.LoraLoaderMixin.load_lora_weights).

Reply:
Please disregard this. I see this is [handled](https://github.com/huggingface/diffusers/pull/3437/files#diff-c54108820590e5247c5a7bd1aa43b68d53a40bc630f0791cb537047752520fabR897).

Reply:
Yep.